ddtelemetry/Cargo.toml

```diff
@@ -6,27 +6,47 @@ version = "0.8.0"
 
 [dependencies]
 anyhow = {version = "1.0"}
+bincode = {version = "1.3"}
```

Comment:
You added a bunch of dependencies that are never used.

Reply:
Sorry, I though I cleaned up most of them. Removed